lib/ledger_sync/ledgers/netsuite/type/deserializer/account.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
```

Comment:
If we follow the naming convention in [the lib root](https://github.com/LedgerSync/ledger_sync/tree/master/lib/ledger_sync/serialization/type), this file would be `lib/ledger_sync/ledgers/netsuite/type/deserializer_account_type.rb`

Reply:
I can change all the NS types to follow this pattern

Reply:
@ryanwjackson I'll leave this as it is for this PR as it follows the existing naming pattern and will follow up with a second PR to adjust all the NS types